packages/flutter_tools/test/general.shard/flutter_validator_test.dart

```diff
@@ -97,9 +105,7 @@ void main() {
       flutterRoot: () => 'sdk/flutter',
     );
 
-    when(flutterVersion.channel).thenReturn('unknown');
-    when(flutterVersion.frameworkVersion).thenReturn('0.0.0');
-    when(flutterVersion.frameworkDate).thenThrow(VersionCheckError('version error'));
+    when(flutterVersion.frameworkCommitDate).thenThrow(VersionCheckError('version error'));
```

Comment:
I'm thinking that for cases like this, it might make sense to create a dedicated fake instead of baking it all in one

Reply:
done